### ISSUE TITLE
Text field limit message translation error

### ISSUE THREAD

**Issue by TomDeSmet** [verbb/formie#420]
**Description**
I have a text field with a limit amount of characters.
Underneath the text field there is the text "x characters left" which I was able to translate by adding `{num} characters left`
to the formie.php translation file. This works, but as soon as I start to type and the `{num}` changes, the translation is gone.

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.3.25
- Craft version: 3.6.11.1
- Multi-site: Yes

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.3.26",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.3.27](https://github.com/verbb/formie/releases/tag/1.3.27)
